### ISSUE TITLE
Why no relu layers in add_extras ssd.py

### ISSUE THREAD

**Issue by notrandom34** [amdegroot/ssd.pytorch#256]
I was wondering why you did not use any relu layers or batchnorm in the add_extras function.

**Comment by d-li14** [comment]
relu does exist in the SSD module here https://github.com/amdegroot/ssd.pytorch/blob/master/ssd.py#L87

**Comment by notrandom34** [comment]
Oh sorry didn't reach that part of code yet lol. Thanks.
